2 - starting acspyexmplClientErrorComponent
2 - Creating ClientErrorComponent
1 - ContainerStatusMsg: Startup begins
1 - ContainerStatusMsg: ORB initialization begins
1 - ContainerStatusMsg: ORB initialization ends
1 - ContainerStatusMsg: Manager access initialization begins
1 - ContainerStatusMsg: Manager access initialization ends
1 - ContainerStatusMsg: Automatic component loading begins
1 - ----------T--:--:--.--- pyErrContainer taggedmessage - Info message from the manager: Startup statistics: 0 components queued to be activated.
1 - ----------T--:--:--.--- pyErrContainer taggedmessage - Info message from the manager: Startup statistics: 0 of 0 components activated.
1 - ContainerStatusMsg: Automatic component loading ends
1 - ContainerStatusMsg: Ready
1 - ----------T--:--:--.--- pyErrContainer __init__ - Container pyErrContainer waiting for requests
1 - ContainerStatusMsg: Startup ends
1 - ----------T--:--:--.--- pyErrContainer activate_component - Activated component: ERRORCOMP_PY
1 - Hello World
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host TestReceiveRemoteException - Example 1: Calls a method that throws an exception.
2 - ErrorTrace (TimeStamp=xxxx,
2 -             File=xxx
2 -             Line=xxx
2 -             Routine=TestReceiveRemoteException,
2 -             Host=xxxx,
2 -             Process=xxx,
2 -             Thread=MainThread,
2 -             Type=10,
2 -             Code=17,
2 -             ShortDescrip=An error occurred, see ErrorDesc member,
2 -             Severity=Error,
2 -             Data: Name=ErrorDesc, Value=badMethod has thrown the expected exception; )
2 - 
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host Error Trace
2 -     ErrorTrace (TimeStamp=xxxx,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host TestReceiveRemoteCompletion - Example 2a: Calls a method that returns an OK completion.
2 -                 File=xxx
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host TestReceiveRemoteCompletion - Completion Ok, without error trace
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host TestReceiveRemoteCompletion - Example 2b: Calls a method that returns an Error completion, of depth 3.
2 -                 Line=xxx
2 -                 Routine=badMethod,
2 -                 Host=xxxx,
2 -                 Process=xxx,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host Error Trace
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host TestReceiveRemoteCompletion - Completion with error trace
2 -                 Thread=omniORB--XXXXXXX,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testExceptionFromCompletion - Example 3a: ExceptionFromCompletion with depth 0
2 - ErrorTrace (TimeStamp=xxxx,
2 -                 Type=10,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testExceptionFromCompletion - OK: No exception thrown
2 -                 Code=17,
2 -             File=xxx
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testExceptionFromCompletion - Example 3b: ExceptionFromCompletion with depth 3
2 -             Line=xxx
2 -                 ShortDescrip=An error occurred, see ErrorDesc member,
2 -             Routine=__errorTrace,
2 -                 Severity=Error,
2 -                 Data: Name=ErrorDesc, Value=Generated multi level exception; )
2 -             Host=xxxx,
2 - 
2 -             Process=xxx,
2 -         ErrorTrace (TimeStamp=xxxx,
2 -             Thread=omniORB--XXXXXXX,
2 -                     File=xxx
2 -             Type=10,
2 -             Code=17,
2 -                     Line=xxx
2 -                     Routine=__errorTrace,
2 -             ShortDescrip=An error occurred, see ErrorDesc member,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testExceptionFromCompletion - OK: ACSErrTypeCommon.GenericErrorEx exception thrown
2 -                     Host=xxxx,
2 -             Severity=Error,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host Error Trace
2 -             Data: Name=ErrorDesc, Value=Generated multi level exception; )
2 - ErrorTrace (TimeStamp=xxxx,
2 -                     Process=xxx,
2 - 
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testTypeException - Example 4a: TypeException with depth 0
2 -             File=xxx
2 -                     Thread=omniORB--XXXXXXX,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testTypeException - Example 4b: TypeException with depth 3
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testTypeException - OK: No exception thrown
2 -     ErrorTrace (TimeStamp=xxxx,
2 -             Line=xxx
2 -                     Type=10,
2 -                     Code=17,
2 -                 File=xxx
2 -             Routine=__returnCompletion,
2 -             Host=xxxx,
2 -                 Line=xxx
2 -                     ShortDescrip=An error occurred, see ErrorDesc member,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host Error Trace
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testTypeException - OK: ACSErrTypeCommonEx exception thrown
2 -             Process=xxx,
2 -                 Routine=__errorTrace,
2 -                     Severity=Error,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testCORBASystemException - Example 5: CORBA exception
2 -                     Data: Name=ErrorDesc, Value=Generated multi level exception; )
2 - ErrorTrace (TimeStamp=xxxx,
2 -                 Host=xxxx,
2 -             Thread=omniORB--XXXXXXX,
2 - 
2 -             File=xxx
2 -                 Process=xxx,
2 -             Type=10,
2 -             Code=17,
2 -             ErrorTrace (TimeStamp=xxxx,
2 -             Line=xxx
2 -                 Thread=omniORB--XXXXXXX,
2 -                         File=xxx
2 -             Routine=typeException,
2 -             ShortDescrip=An error occurred, see ErrorDesc member,
2 -                 Type=10,
2 -                 Code=17,
2 -             Host=xxxx,
2 -                         Line=xxx
2 -             Severity=Error,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testCORBASystemException - OK: CORBA.SystemException exception received
2 -             Data: Name=ErrorDesc, Value=Exception generated by adding an error trace from a completion; )
2 -             Process=xxx,
2 -                         Routine=__errorTrace,
2 -                 ShortDescrip=An error occurred, see ErrorDesc member,
2 - 
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host Error Trace
2 - ErrorTrace (TimeStamp=xxxx,
2 -                         Host=xxxx,
2 -                 Severity=Error,
2 -             Thread=omniORB--XXXXXXX,
2 -                 Data: Name=ErrorDesc, Value=Generated multi level exception; )
2 -     ErrorTrace (TimeStamp=xxxx,
2 -             File=xxx
2 -                         Process=xxx,
2 -             Type=10,
2 - 
2 -             Code=17,
2 -                 File=xxx
2 -             Line=xxx
2 -                         Thread=omniORB--XXXXXXX,
2 -         ErrorTrace (TimeStamp=xxxx,
2 -                 Line=xxx
2 -             Routine=testCORBASystemException,
2 -             ShortDescrip=An error occurred, see ErrorDesc member,
2 -                         Type=10,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testCompletionFromCompletion - Example 6a: completionFromCompletion with depth 0
2 -                         Code=17,
2 -                     File=xxx
2 -             Host=xxxx,
2 -                 Routine=__returnCompletion,
2 -             Severity=Error,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testCompletionFromCompletion - Completion Ok, without error trace
2 -             Data: )
2 -                 Host=xxxx,
2 -                     Line=xxx
2 -             Process=xxx,
2 -                         ShortDescrip=An error occurred, see ErrorDesc member,
2 - 
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testCompletionFromCompletion - Example 6b: completionFromCompletion with depth 3
2 -                 Process=xxx,
2 -                     Routine=__errorTrace,
2 -                         Severity=Error,
2 -             Thread=MainThread,
2 -                         Data: Name=ErrorDesc, Value=Bottom of error trace; )
2 -     ErrorTrace (TimeStamp=xxxx,
2 -                     Host=xxxx,
2 -                 Thread=omniORB--XXXXXXX,
2 -             Type=10,
2 - 
2 -             Code=17,
2 -                 File=xxx
2 -                     Process=xxx,
2 -                 Type=10,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host Error Trace
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testCompletionFromCompletion - OK: Completion with error trace
2 -                 Code=17,
2 -                 Line=xxx
2 -             ShortDescrip=An error occurred, see ErrorDesc member,
2 -                     Thread=omniORB--XXXXXXX,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testOutCompletion - Example 8: outCompletion
2 - ErrorTrace (TimeStamp=xxxx,
2 -                 Routine=__errorTrace,
2 -             Severity=Error,
2 -                 ShortDescrip=An error occurred, see ErrorDesc member,
2 -                     Type=10,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host testOutCompletion - Completion Ok, without error trace
2 -                     Code=17,
2 -             Data: )
2 -             File=xxx
2 -                 Host=xxxx,
2 -                 Severity=Error,
2 - 
2 -                 Data: )
2 -             Line=xxx
2 -                 Process=xxx,
2 -                     ShortDescrip=An error occurred, see ErrorDesc member,
1 - ----------T--:--:--.--- pyErrContainer deactivate_component - Deactivating component: ERRORCOMP_PY
2 - 
2 -     ErrorTrace (TimeStamp=xxxx,
2 -             Routine=__returnCompletion,
2 -                     Severity=Error,
2 -                 Thread=omniORB--XXXXXXX,
1 - Goodbye
2 -                     Data: Name=ErrorDesc, Value=Bottom of error trace; )
2 -         ErrorTrace (TimeStamp=xxxx,
2 -                 File=xxx
2 -             Host=xxxx,
2 -                 Type=10,
2 - 
2 -                 Code=17,
2 -                     File=xxx
2 -                 Line=xxx
2 -             Process=xxx,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host disconnect - Disconnect called for client
2 -                     Line=xxx
2 -                 Routine=testCORBASystemException,
2 -                 ShortDescrip=An error occurred, see ErrorDesc member,
2 -             Thread=omniORB--XXXXXXX,
2 -                 Host=xxxx,
2 -                     Routine=__returnCompletion,
2 -                 Severity=Error,
2 -             Type=10,
2 - ----------T--:--:--.--- acspyexmplClientErrorComponent initialized by user@host disconnect - Disconnect called for client
2 -             Code=17,
2 -                 Data: Name=ErrorDesc, Value=Generated multi level exception; )
2 -                     Host=xxxx,
2 -                 Process=xxx,
2 - 
2 -                     Process=xxx,
2 -             ShortDescrip=An error occurred, see ErrorDesc member,
2 -                 Thread=MainThread,
2 -         ErrorTrace (TimeStamp=xxxx,
2 -             Severity=Error,
2 -                     Thread=omniORB--XXXXXXX,
2 -                 Type=7,
2 -                 Code=0,
2 -             Data: Name=ErrorDesc, Value=Put an error trace in completionFromCompletion; )
2 -                     File=xxx
2 -                     Type=10,
2 - 
2 -                     Code=17,
2 -                     Line=xxx
2 -                 ShortDescrip=A Python exception,
2 -     ErrorTrace (TimeStamp=xxxx,
2 -                     Routine=__errorTrace,
2 -                 Severity=Error,
2 -                     ShortDescrip=An error occurred, see ErrorDesc member,
2 -                 File=xxx
2 -                     Host=xxxx,
2 -                     Severity=Error,
2 -                     Data: )
2 -   File "/somepath/acspyexmplClientErrorComponent, line XXX, in testCORBASystemException
2 -                 Line=xxx
2 -                     Process=xxx,
2 - 
2 -                 Routine=__returnCompletion,
2 -     self.foo.corbaSystemException()
2 -                     Thread=omniORB--XXXXXXX,
2 -   File "/somepath/acsexmplErrorComponent_idl.py, line XXX, in corbaSystemException
2 -                 Host=xxxx,
2 -                     Type=10,
2 -                     Code=17,
2 -                 Process=xxx,
2 -     return self._obj.invoke("corbaSystemException", _0_acsexmplErrorComponent.ErrorComponent._d_corbaSystemException, args)
2 - BAD_PARAM: CORBA.BAD_PARAM(0x0, CORBA.COMPLETED_NO)
2 -                     ShortDescrip=An error occurred, see ErrorDesc member,
2 -                 Thread=omniORB--XXXXXXX,
2 - ; )
2 -                     Severity=Error,
2 -                 Type=10,
2 - 
2 -                 Code=17,
2 -                     Data: Name=ErrorDesc, Value=Bottom of error trace; )
2 - 
2 -                 ShortDescrip=An error occurred, see ErrorDesc member,
2 -                 Severity=Error,
2 -                 Data: )
2 - 
2 -         ErrorTrace (TimeStamp=xxxx,
2 -                     File=xxx
2 -                     Line=xxx
2 -                     Routine=__returnCompletion,
2 -                     Host=xxxx,
2 -                     Process=xxx,
2 -                     Thread=omniORB--XXXXXXX,
2 -                     Type=10,
2 -                     Code=17,
2 -                     ShortDescrip=An error occurred, see ErrorDesc member,
2 -                     Severity=Error,
2 -                     Data: )
2 - 
1 - ----------T--:--:--.--- INFO [acsStartContainer] Container: 'pyErrContainer' exited with code: 0.
